Resources/config/yui.yml

```diff
@@ -1078,3 +1078,36 @@ system:
                 ez-app-extension-admin:
                     requires: ['ez-serversideview', 'ez-serversideviewservice', 'ez-app-extension']
                     path: %ez_platformui.public_dir%/js/apps/extensions/ez-app-extension-admin.js
+                ez-userprofileview:
+                    requires:
+                        - 'ez-templatebasedview'
+                        - 'event'
+                        - 'event-tap'
```

Comment:
When you are adding `event` module as a dependency, then you don't have to add the `event-tap` module.
The `event` module already contains `event-tap` module.
http://yuilibrary.com/yui/docs/api/modules/event.html

BTW. The `event` module is a big module, do you need it here?